sphinx/sphinx.gradle

```diff
@@ -23,12 +23,22 @@ def copySphinxLib = tasks.register('copySphinxLib', Sync) {
 def sphinxDockerfile = tasks.register('sphinxDockerfile', Dockerfile) {
     destFile.set layout.buildDirectory.file('sphinx-image/Dockerfile')
     // Deephaven server python API requires that the wheel be installed to build, so we share this image
-    from 'deephaven/runtime-base:local-build'
+    from 'deephaven/server:local-build'
 
     copyFile "./wheel", "/wheel"
     copyFile "./lib", '${VIRTUAL_ENV}/lib/python3.7/site-packages/'
 
+    environmentVariable 'DEEPHAVEN_CLASSPATH', '/opt/deephaven/server-0.10.0/lib/*:/opt/deephaven/server-0.10.0/lib'
+    environmentVariable 'DEEPHAVEN_WORKSPACE', '/tmp/workspace'
+    environmentVariable 'DEEPHAVEN_DEVROOT', '/tmp/devroot'
+    environmentVariable 'DEEPHAVEN_PROPFILE', 'dh-defaults.prop'
+    environmentVariable 'JDK_HOME', '/usr/lib/jvm/zulu8/jre/'
+    environmentVariable 'JAVA_VERSION', '1.8'
```

Comment:
I don't think these environment variables are true wrt deephaven/server. If they aren't actually used / important, we should see if we can delete them?